css/style.css

```diff
@@ -72,7 +72,7 @@ header {
             margin-right: 10px; }
 
 .hero {
-  background: #313234 url("https://static.pexels.com/photos/6063/laptop-notebook-technology-computer.jpg") no-repeat 0/cover;
+  background: #313234 url("/images/after/laptop-notebook-technology-computer.jpg") no-repeat 0/cover;
```

Comment:
ustawiasz adresy obrazkow rozpoczynajac od katalogu glownego na serwerze (slash na poczatku), co nie jest uniwersalne. Jak otworze sam index.html (nie przez browsersync) to obrazkow nie zobacze.